ui/apps/platform/src/Containers/Policies/PatternFly/Wizard/Step5/ReviewPolicyForm.tsx

```diff
@@ -160,6 +160,16 @@ function ReviewPolicyForm({
                         <div className="pf-u-mb-md">
                             Before you save the policy, verify that the violations seem accurate.
                         </div>
+                        {values.disabled && (
+                            <Alert
+                                className="pf-u-mb-md"
+                                isInline
+                                variant="info"
+                                title="Policy disabled"
+                            >
+                                <p>Violations will not generate unless policy is enabled</p>
```

Comment:
```suggestion
                                <p>Violations will not be generated unless the policy is enabled</p>
```

Reply:
It would be _really_ nice if this also came with a link that allowed people to enable the policy (or took them to the screen where they can), but that requires more intricate JS than it's probably worth...

Reply:
Yeah, I think adding a button there to enable the policy could be a separate UI task.